public/assets/scripts/choices.js

```diff
@@ -4880,17 +4887,11 @@
             var blurWasWithinContainer = target && containerOuter.element.contains(target);
             if (blurWasWithinContainer && !this._isScrollingOnIe) {
                 var targetIsInput = target === this.input.element;
-                if (this._isTextElement || this._isSelectMultipleElement) {
-                    if (targetIsInput) {
-                        containerOuter.removeFocusState();
-                        this.hideDropdown(true);
-                        this.unhighlightAll();
-                    }
-                }
-                else {
+                if (targetIsInput) {
                     containerOuter.removeFocusState();
-                    if (targetIsInput || (target === containerOuter.element && !this._canSearch)) {
-                        this.hideDropdown(true);
+                    this.hideDropdown(true);
+                    if (this._isTextElement || this._isSelectMultipleElement) {
+                        this.unhighlightAll();
                     }
                 }
```

Comment:
Whilst this works for the search not being hidden, we now don't take into account to remove the focusState on the previous outerElement when tabbing through. The previous element won't lose their `is-focused` class.

We'd actually need to check for another condition afterwards to remove the previous class:


```js
if (target === this.input.elemen) {
   // your logic
}
// Remove the focus state when the past outerContainer was the target
else if (target === this.containerOuter.element) {
  containerOuter.removeFocusState();
}
```

Reply:
I've restored that branch, and compiled the js file. Want to give that a test to see if that encounters any other regressions or accessibility issues?

Reply:
Already did - the change looks good to me :)